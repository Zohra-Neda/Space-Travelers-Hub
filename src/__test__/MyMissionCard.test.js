import React from 'react';
import renderer from 'react-test-renderer';
import MyMissionCard from '../components/MyMissionCard';

describe('MyMissionCard component', () => {
  test('renders correctly', () => {
    const mockMission = {
      id: 'mission1',
      name: 'Mission 1',
      description: 'This is the mission description',
      joined: true,
    };

    const component = renderer.create(<MyMissionCard mission={mockMission} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
