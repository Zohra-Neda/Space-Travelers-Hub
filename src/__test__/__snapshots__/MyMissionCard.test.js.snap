// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyMissionCard component renders correctly 1`] = `
<div
  className="my-mission-card border border-1  text-2xl ps-4 py-5 "
>
  Mission 1
</div>
`;
